raiden-ts/src/raiden.ts

```diff
@@ -1139,100 +1144,37 @@ export class Raiden {
    * @param options - tx options
    * @param options.subkey - By default, if using subkey, main account is used to send transactions
    *    Set this to true if one wants to force sending the transaction with the subkey
-   * @param options.allowance - Value to approve on service token. Allows further deposits without
-   *    needing to approve again. Defaults to exact amount.
    * @returns transaction hash
    */
   public async depositToUDC(
     amount: BigNumberish,
     onChange?: OnChange<EventTypes, { txHash: string }>,
-    { subkey, allowance }: { subkey?: boolean; allowance?: BigNumberish } = {},
+    { subkey }: { subkey?: boolean } = {},
```

Comment:
We're not yet constrained to backwards compatibility, and this was a very very edgy feature which I'm 100% sure no one of the early adopters used, so I took the freedom to remove it since it isn't relevant anymore with the default `minimumAllowance`, and the user can always edit the allowance on Metamask prompt anyway

Reply:
Actually I meant that we still have an object `options: { subkey?: boolean }` instead of just `subkey = false`....

Reply:
Oh, but `subkey = false` is a different behavior than `undefined`: The on-chain txs can be performed by the main key or the subkey, depending on the `config.subkey` value (defaults to using the main account when possible, but can be changed to `true` to use the subkey). If you pass `false` here, only for this operation, it'll force use the mainkey regardless of the `config.subkey`.

Reply:
Well then `subkey?: boolean`. Doesn't matter.
Anyways. Out of focus now.